app/views/cards/_card.html.erb

```diff
@@ -0,0 +1,12 @@
+<tr class="tr-padding-top status-needs-review">
+  <th><%= link_to card.id, edit_deck_card_path(@deck, card) %></th>
```

Comment:
we should get this to link specifically to the correct form for the current card type